xml/System.Security.AccessControl/ObjectSecurity.xml

```diff
@@ -1306,7 +1306,14 @@
         <param name="preserveInheritance">
           <see langword="true" /> to preserve inherited access rules; <see langword="false" /> to remove inherited access rules. This parameter is ignored if <paramref name="isProtected" /> is <see langword="false" />.</param>
         <summary>Sets or removes protection of the access rules associated with this <see cref="T:System.Security.AccessControl.ObjectSecurity" /> object. Protected access rules cannot be modified by parent objects through inheritance.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks
+When you call the method with `isProtected=true` and `preserveInheritance=true`, you need to walk the new ACL of the object and check for DENY type ACEs. For a canonically sorted DACL, the DENY ACEs must appear in the front of the DACL.
```

Comment:
Added newlines after each sentence (which doesn't create new paragraphs, just makes the markdown more manageable), and added a variant of Herbert's suggested extra sentence.

```suggestion
When you call the method with `isProtected=true` and `preserveInheritance=true`, you need to walk the new ACL of the object and check for DENY type ACEs.
For a canonically sorted DACL, the DENY ACEs must appear in the front of the DACL.
For more information on the canonical ordering of ACLs, see [Order of ACEs in a DACL](https://docs.microsoft.com/en-us/windows/win32/secauthz/order-of-aces-in-a-dacl).
```

Reply:
@HerbertMauerer can you address this suggestion so we can get the change merged?